### PR TITLE
Fix all copyright warnings.

### PR DESCRIPTION
**Pull request by tomas-langer**
Fix all copyright warnings. The plugin will be modified to fail on wrong number of lines of a copyright.
Kindly update you templates!

### CODE REVIEW

**Review by tjquinno — CHANGES_REQUESTED** [review]
There are formatting issues with some files. Please see notes.

File: jersey/client/src/main/java/io/helidon/jersey/client/ClientBuilderListener.java [hunk 1]
```diff
@@ -12,7 +12,6 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- *
  */
```
[the remark this answers is not in the document]
Comment (tomas-langer) [reply jersey/client/src/main/java/io/helidon/jersey/client/ClientBuilderListener.java:16]: I have done a search and replace that probably did that (and caused some of the formatting failures). I will revisit the modified files (and fix the end of lines in all of them)

File: examples/microprofile/bean-validation/src/test/java/io/helidon/tests/integration/bean/validation/TestValidationEndpoint.java [hunk 2]
```diff
@@ -12,9 +12,7 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- *
- */
-package io.helidon.tests.integration.bean.validation;
+ */package io.helidon.tests.integration.bean.validation;
```
[the remark this answers is not in the document]
Comment (tomas-langer) [reply examples/microprofile/bean-validation/src/test/java/io/helidon/tests/integration/bean/validation/TestValidationEndpoint.java:15]: Should be now fixed in all of them.

File: jersey/client/src/main/java/io/helidon/jersey/client/ClientBuilderListener.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (tomas-langer) [reply jersey/client/src/main/java/io/helidon/jersey/client/ClientBuilderListener.java:16]: I will not fix the extra blank page (or lack thereof), as that impacts hundreds of files (and is not validated anyway).

**Review by tjquinno — APPROVED** [review]

**Review by danielkec — APPROVED** [review]

**Pull request merged**